src/index.html

```diff
@@ -5,6 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fira+Mono:wght@500;700&family=VT323&display=swap"
```

Comment:
Did you try using these? Are you sure this worked?

Reply:
They are working for me. Is this a reminder to test? Or did they not work when you tried them?

Reply:
I didn't try them. Just wanted to make sure that you did!